src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -88,60 +76,72 @@ export const ProjectListItemCard: FC<ProjectListItemCardProps> = ({
   } = useStyles();
 
   return (
-    <Card className={card}>
-      <div className={cardBody}>
-        <CardMedia
-          className={media}
-          image={projectImagePath}
-          title="Project Image"
-        />
-        <CardContent className={cardContent}>
-          <div className={leftContent}>
-            <Typography variant="h4">{name}</Typography>
-            <Typography color="primary" variant="body2">
-              <Box
-                component="span"
-                m={1}
-                display="inline"
-                color="text.secondary"
+    <div className={className}>
```

Comment:
I don't think an extra div should be used here. Either apply the className to Card, or wrap this component with the div where it is used. Separation of concerns.